data/users/usersDataModel.js

```diff
@@ -1,14 +1,15 @@
+
 const db = require("../dbConfig.js");
 
 module.exports = {
     register,
     login
 }
 
-function register() {
-
+function register(userData) {
+    return db("users").insert({"username": userData.username, "password": userData.password});
```

Comment:
this will need to be hashed right?

Reply:
Hi @kamry-bowman - thanks for checking in . I actually am hashing it in the user Router!

I am actually having issues parsing it myself and I'd like to see if you could be an extra pair of eyes to figure out where the parsing issues are coming from!